config.toml

```diff
@@ -25,4 +25,4 @@ pg_params = ""
 log_sql_queries = true
 
 [content]
-path = "/rules_content"
```

Comment:
Please add the same into confi-devel.toml as well. Besides  environment variables should be updated here https://github.com/RedHatInsights/e2e-deploy/blob/master/templates/ccx-data-pipeline/insights-results-aggregator.yml#L72

Reply:
Updated the configs file and created a PR in e2e-deploy
https://github.com/RedHatInsights/e2e-deploy/pull/1215